docs/manufacturing.md

```diff
@@ -1,13 +1,67 @@
 # Babel-Babel manufacturing
 
+## PCB specification for all PCB
+### Main PCB
+* Quantity per product/order: 1
+* Layers: 4 (layer stack in manufacturing package)
+* Thickness: 1.0mm
+* PCB color: Red
+* Silkscreen: White
+* Surface Finish: LeadFree HASL-RoHS
+* Outer Copper Weight: 1 oz or 1.4 Mils
+* Inner Copper Weight: 0.5 oz or 0.7 mils
+* Material Type: FR4-Standard Tg 130-140C
+* Flying Prob Test: Fully Test
+* Castellated Holes: No
+
+### Enclosure PCB
+
+### Part B1
+   * Quantity per product/order: 1
+   * Thickness: 0.8mm
+   * Surface Finish: LeadFree HASL-RoHS
+   * Color: Red
+   * Silkscreen: White
+   * Material: Textolite
```

Comment:
I don't think this requirement is meaningful as it is unnecessarily limiting. I would recommend stating simply "FR4 or similar". This comment also applies to the following sections.